packages/itwin/reports-config-widget/api/reports-config-widget-react.api.md

```diff
@@ -0,0 +1,129 @@
+## API Report File for "@itwin/reports-config-widget-react"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+/// <reference types="react" />
+
+import type { AccessToken } from '@itwin/core-bentley';
+import { ExtractionClient } from '@itwin/insights-client';
+import { IModelsClient } from '@itwin/imodels-client-management';
+import type { Localization } from '@itwin/core-common';
+import { MappingsClient } from '@itwin/insights-client';
+import { default as React_2 } from 'react';
+import type { Report } from '@itwin/insights-client';
+import { ReportsClient } from '@itwin/insights-client';
+import { StagePanelLocation } from '@itwin/appui-react';
+import { StagePanelSection } from '@itwin/appui-react';
+import type { UiItemsProvider } from '@itwin/appui-react';
+import type { Widget } from '@itwin/appui-react';
+
+// @public (undocumented)
+export class BulkExtractor {
+    constructor(reportsClient: ReportsClient, extractionClient: ExtractionClient, getAccessToken: () => Promise<AccessToken>, successfulExtractionToast: (iModelName: string, odataFeedUrl: string) => void, failedExtractionToast: (iModelName: string) => void);
+    // (undocumented)
+    getIModelState(iModelId: string, iModelName: string, odataFeedUrl: string): Promise<ExtractionStates>;
+    // (undocumented)
+    getReportState(reportId: string): Promise<ExtractionStates>;
+    // (undocumented)
+    runIModelExtraction(iModelId: string): Promise<void>;
+    // (undocumented)
+    runIModelExtractions(iModels: string[]): Promise<void>;
+    // (undocumented)
+    runReportExtractions(reportIds: string[]): Promise<void>;
+    // (undocumented)
+    setHook(setJobRunning: React.Dispatch<React.SetStateAction<boolean>>, iModels: string[]): void;
+}
+
+// @public
+export const ReportAction: ({ report, onSaveSuccess, onClickCancel }: ReportActionProps) => JSX.Element;
+
+// @public
+export interface ReportActionProps {
+    // (undocumented)
+    onClickCancel?: () => void;
+    // (undocumented)
+    onSaveSuccess: () => void;
+    // (undocumented)
+    report?: Report;
+}
+
+// @public
+export const ReportMappings: ({ report, onClickClose, defaultIModelId }: ReportMappingsProps) => JSX.Element | null;
+
+// @public
+export interface ReportMappingsProps {
+    // (undocumented)
+    defaultIModelId?: string;
+    // (undocumented)
+    onClickClose: () => void;
+    // (undocumented)
+    report: Report;
+}
+
+// @public
+export const Reports: ({ onClickAddReport, onClickReportModify, onClickReportTitle, }: ReportsProps) => JSX.Element | null;
+
+// @public (undocumented)
+export const REPORTS_CONFIG_BASE_URL = "https://api.bentley.com";
```

Comment:
I didn't see the base url exposed for other widgets.

Reply:
G&M widget does not need it yet.. EC3 does not have it defined anywhere, you can define it on your own. I don't even think its worth exporting this. What do you think?

Reply:
Yeah, I don't think it's needed. For now mark it as internal if it is already exposed.

Reply:
Done.